pkg/kv/params/database.go

```diff
@@ -10,12 +10,27 @@ type KV struct {
 	DynamoDB *DynamoDB
 }
 
+// EnableMetrics returns a copy of params with Metrics enabled for Postgres.
+// This is used only when we like to get parameters for the main run of lakeFS
+// as the is only one metrics set we expose
+func (p KV) EnableMetrics() KV {
```

Comment:
I don't think this should be here, it's specific logic for postgres.
We should define the parameter as bool pointer in the config.go struct.
On GetKVParams we should check if it's nil and assign it the default value

Reply:
Didn't want the metrics we expose to be optional or configurable in the application configuration.
Needed a way to configure the metrics registration for a specific pgx pool.
While the process is running there are cases, like while performing 'setup', that we open more than one kv store.
In this case we need to make sure that the initial connection database pool will be the only one that reports these metrics.